vendor/extensions/reports/app/views/refinery/reports/show.html.erb

```diff
@@ -7,7 +7,9 @@
         <div class="content__date"><%= @report.date.strftime("%B %d, %Y") %></div>
         <div class="content__tags">Tags: <%= @tags %></div>
         <div class="content__body">
-          <img src="<%= @image_url %>" class="content__image" />
+        <% if @report.image %>
+          <img src="<%= @report.image.url %>" class="content__image" />
```

Comment:
Did you consider trying an ERB image_tag? https://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag